runltp-ng

```diff
@@ -64,7 +68,9 @@ if ($help) {
 	print("--list             : List test groups\n\n");
 	print("--verbose          : Enables verbose mode\n\n");
 	print("--run=testgroup    : Execute group of tests\n\n");
+    print("--interactive      : Print command to start interactive session with SUT\n\n");
```

Comment:
With `--interactive`, I would expect something differently. For me it is more `--dry-run`, wdyt?

Reply:
It is not a dry-run though, it is just printing a command you can use to start the VM. Ideally it would actually start the VM, but this would require significant changes I think.